folly/memory/UninitializedMemoryHacks.h

```diff
@@ -110,6 +130,11 @@ inline void resizeWithoutInitialization(
       s.reserve(n);
     }
     detail::unsafeStringSetLargerSize(s, n);
+#if defined(_MSC_VER) && _MSC_VER >= 1938 && defined(_INSERT_STRING_ANNOTATION)
+    if (_Asan_string_should_annotate && s.capacity() > std::string().capacity())
```

Comment:
Where does the symbol `_Asan_string_should_annotate` come from?

Reply:
I looked into the code of std::string how they do it and replicated code pretty much